html2text.php

```diff
@@ -224,11 +224,6 @@ public function fix_image_paths($input, $contextid, $filearea, $itemid, $kfactor
                         // Add filename to list of temporary files.
                         $this->tempfiles[] = $file;
 
-                        // In answer texts we want a line break to avoid the picture going above the line.
-                        if ($filearea == 'answer') {
-                            $output .= '<br/>';
```

Comment:
deleting this leads to sometimes not readable formulas, that are above the line. That's why there is always a newline after formulas. So solving your problem leads to another problem :( We might want to make a preference, so everyone can choose which works best for him?